webstruct/text_tokenizers.py

```diff
@@ -101,6 +101,11 @@ class WordTokenizer(object):
     [TextToken(chars='``', position=0, length=1),
      TextToken(chars='a', position=2, length=1)]
 
+    >>> WordTokenizer().segment_words('["a')
+    [TextToken(chars='[', position=0, length=1),
+     TextToken(chars='``', position=1, length=1),
+     TextToken(chars='a', position=2, length=1)]
```

Comment:
Could you please update the docstring of WorkTokenizer? It says that it is just a copy-pasted version of http://www.nltk.org/_modules/nltk/tokenize/treebank.html, but it is much more now.

Reply:
Updated description adding mentioning of custom span_tokenize method